Patches/GameOptionsMenuPatch.cs

```diff
@@ -39,13 +43,76 @@ public static class GameSettingMenuInitializeOptionsPatch
 {
     public static void Prefix(GameSettingMenu __instance)
     {
+
         // Unlocks map/impostor amount changing in online (for testing on your custom servers)
         // Changed to be able to change the map in online mode without having to re-establish the room.
         __instance.GameSettingsTab.HideForOnline = new Il2CppReferenceArray<Transform>(0);
     }
     // Add Dleks to map selection
-    public static void Postfix()
+    public static void Postfix(GameSettingMenu __instance)
     {
+        var gamepreset = __instance.GamePresetsButton;
+
+        var gamesettings = __instance.GameSettingsButton;
+        __instance.GameSettingsButton.transform.localScale = new Vector3(0.5f, 0.5f, 1f);
+        __instance.GameSettingsButton.transform.localPosition = new Vector3(gamesettings.transform.localPosition.x, gamepreset.transform.localPosition.y + 0.2f, gamesettings.transform.localPosition.z);
+
+        var rolesettings = __instance.RoleSettingsButton;
+        __instance.RoleSettingsButton.transform.localScale = new Vector3(0.5f, 0.5f, 1f);
+        __instance.RoleSettingsButton.transform.localPosition = new Vector3(rolesettings.transform.localPosition.x, gamesettings.transform.localPosition.y - 0.4f, rolesettings.transform.localPosition.z);
+        //rolesettings.OnClick.RemoveAllListeners();
+        // button.OnClick.AddListener( () => {}); // add rolemenu method
+
+        //button 1
+        GameObject template = gamepreset.gameObject;
+        GameObject targetBox = UnityEngine.Object.Instantiate(template, gamepreset.transform);
+        targetBox.name = "System Settings";
+        targetBox.transform.localScale = new Vector3(0.6f, 0.6f, 1f);
+        targetBox.transform.localPosition = new Vector3(targetBox.transform.localPosition.x + 2.95f, rolesettings.transform.localPosition.y - 0.1f, targetBox.transform.localPosition.z);
+
+        _ = new LateTask(() =>
+        {
+            targetBox.transform.parent = null;
+            // gamepreset.transform.localScale = new Vector3(0f, 0f, 0f);
+            gamepreset.gameObject.SetActive(false);
+            targetBox.transform.parent = __instance.transform.Find("LeftPanel");
+        }, 0f); // remove GamePresets
```

Comment:
Wait, why do you use LateTask if it is completely useless?

Reply:
for some reason, it requires for it to be delayed a tiny bit. or else the text wouldn't change.

Reply:
0f gives the smallest delay

Reply:
Hmm, ok, but need give name for this LateTask